storage/engine/db.cc

```diff
@@ -27,9 +27,9 @@
 #include "rocksdb/options.h"
 #include "rocksdb/table.h"
 #include "rocksdb/utilities/write_batch_with_index.h"
-#include "cockroach/proto/api.pb.h"
-#include "cockroach/proto/data.pb.h"
-#include "cockroach/proto/internal.pb.h"
+#include "proto/api.pb.h"
+#include "proto/data.pb.h"
+#include "proto/internal.pb.h"
```

Comment:
I'm a bit confused. It looks to me like the files are still stored under `cockroach/proto`. How is this working?